message/message-factory.rst

```diff
@@ -1,57 +1,124 @@
 .. _message-factory:
+.. _stream-factory:
 
-Message Factory
-===============
+HTTP Factories
```

Comment:
Although I agree that they are not really Message Factories in most places we refer to them as message factories. The file name also says message-factory, so this could be a little bit confusing.